modules/@apostrophecms/migration/lib/addMissingSchemaFields.js

```diff
@@ -54,8 +90,10 @@ module.exports = (self) => {
     addMissingSchemaFieldsFor(doc, schema, dotPath, changes) {
       for (const field of schema) {
         const newDotPath = dotPath ? `${dotPath}.${field.name}` : field.name;
-        // Supply the default
-        if (doc[field.name] === undefined) {
+        // Supply the default if a field is undefined, and also in the
```

Comment:
Not really our fault per se if the object field had a weird non-object value rather than being undefined, but it happened "in the field" with testbed, so fix it!